security/security-secrets-setup.1.rst

```diff
@@ -0,0 +1,191 @@
+NAME
+====
+
+security-secrets-setup — Creates an on-device public-key infrastructure (PKI) to secure microservice secret management
+
+SYNOPSIS
+========
+
+| security-secrets-setup [-confdir confdir] -config params.json
+| security-secrets-setup [-confdir confdir] -generate
+| security-secrets-setup [-confdir confdir] -cache
+| security-secrets-setup [-confdir confdir] -import
+| security-secrets-setup [-h|--help]
+
+
+DESCRIPTION
+===========
+
+The Vault secret management component of EdgeX Foundry requires TLS encryption
+of secrets over the wire via a pre-created PKI.
+security-secrets-setup is responsible for creating a certificate authority
+and any needed TLS leaf certificates in order to secure the EdgeX security services.
+security-secrets-setup supports several modes of operation as defined in the OPTIONS section.
+
+As the PKI is security-sensitive,
+this tool takes a number of precautions to safeguard the PKI:
+
+* The PKI can be deployed to transient storage to address potential attacks to the PKI at-rest.
+
+* The PKI is deployed such that each service has its own assets folder, which is amenable to security controls imposed by container runtimes such as mandatory access controls or file system namespaces.
+
+* The private key of the certificate authority (CA) is shredded (securely erased) prior to caching or deployment to block issuance of new CA descendants (this is most relevant in caching mode).
+
+
+OPTIONS
+=======
+
+Modes of operation
+------------------
+
+  \-config *params.json*
```

Comment:
I think since we already are breaking so much for Fuji with respect to security configuration, names, options ,etc. we should drop legacy options and try to match as closely as possible with other edgex-go services

Reply:
Legacy -config option dropped for documentation.